maintainers/maintainer-list.nix

```diff
@@ -11980,4 +11980,10 @@
     github = "zupo";
     githubId = 311580;
   };
+  kranzes = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.